### ISSUE TITLE
Out of memory while running demo

### ISSUE THREAD

**Issue by vikram-gupta** [eldar/deepcut-cnn#19]
I am getting out of memory error while running the demo using the following command. I am using GeForce GTX 1070.

`python ./pose_demo.py image.png --out_name=prediction`

```
INFO:estimate_pose:Done!
F1028 23:43:02.663663 10368 syncedmem.cpp:56] Check failed: error == cudaSuccess (2 vs. 0)  out of memory
*** Check failure stack trace: ***
Aborted (core dumped)
```

**Comment by CamilleMaurice** [comment]
Same for me (but not same graphic card). Did you manage to fix it ?

**Comment by CamilleMaurice** [comment]
@vikram-gupta Same for me (but not same graphic card). Did you manage to fix it ?

**Comment by vikram-gupta** [comment]
@CamilleMaurice not yet !

**Comment by CamilleMaurice** [comment]
I've reinstalled it from scratch but I still face the same problem. If I find out a solution I'll let you know here. Btw, I am using anaconda2

**Comment by eldar** [comment]
Hi,

This is interesting. What is the size of image.png? If it is too big, then unfortunately it won't fit into memory.

Cheers,
Eldar.

**Comment by CamilleMaurice** [comment]
The size is 274x512 pixels in the demo. I've tried to resize it to a smaller size, without success. I've already installed deeplearning frameworks using Caffe and Cuda and this is the first time I face this error. I'm not using Debian Wheezy but Ubuntu 16.04 and I am using CUDA 8.0.61

**Comment by asnt** [comment]
Same problem on a 4GB Quadro M2000M. The problem persists whatever the input image size.
Do we need a 12GB card for running on GPU?

**Comment by CamilleMaurice** [comment]
Reading the paper I've also found out the use of 12GB GPU. Next week I might be able to try with 11GB GPU card. I wont have access to more GPU for now so that will be my last test and I'll report here my results.

**Comment by CamilleMaurice** [comment]
Here are some news, I managed to install it and it worked on the : GeForce GTX 1080 Ti 11Go.

**Comment by williammm001** [comment]
It doesn't work on RTX 2080 Super. I guess it might need a 11GB GPU card like 1080Ti.

**Comment by CamilleMaurice** [comment]
> It doesn't work on RTX 2080 Super. I guess it might need a 11GB GPU card like 1080Ti.

Yes. The need for NVIDIA and CUDA is widespread. You often need them to run pose estimation algorithms. It is the same for OpenPose (you might be insterested).

**Comment by williammm001** [comment]
> > It doesn't work on RTX 2080 Super. I guess it might need a 11GB GPU card like 1080Ti.
> 
> Yes. The need for NVIDIA and CUDA is widespread. You often need them to run pose estimation algorithms. It is the same for OpenPose (you might be insterested).

Thanks. I used deepcut just because it is part of the MuVS. I just wonder why it requires so much GPU memory...
